<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GUI_BookInfo</class>
 <widget class="QDialog" name="GUI_BookInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>843</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Информация об издании</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="1">
    <widget class="QLabel" name="headLabel">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <kerning>false</kerning>
      </font>
     </property>
     <property name="text">
      <string>Информация об издании</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="3" rowspan="5">
    <widget class="QWidget" name="allTableWidget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QTableWidget" name="bookInfoTable">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::SingleSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>60</number>
        </attribute>
        <attribute name="horizontalHeaderHighlightSections">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderHighlightSections">
         <bool>false</bool>
        </attribute>
        <column>
         <property name="text">
          <string>ID копии</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>В архиве</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Выдан</string>
         </property>
        </column>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="editCopiesButtons" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="spacing">
          <number>6</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="archieveBookButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>В архив</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="renewBookButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Продлить</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="getBackBookButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Вернуть</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="2">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="1">
    <widget class="QWidget" name="mainButtonsWidget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>86</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="editInfoButton">
        <property name="text">
         <string>Редактировать</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QDialogButtonBox" name="editModeButtons">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="1">
      <widget class="QLabel" name="nameMUST">
       <property name="styleSheet">
        <string notr="true">color : red;</string>
       </property>
       <property name="text">
        <string notr="true">*</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QSpinBox" name="pagesSpinBox">
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="maximum">
        <number>100000</number>
       </property>
       <property name="value">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="authorLabel">
       <property name="text">
        <string>Автор(-ы):</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QSpinBox" name="yearSpinBox">
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="maximum">
        <number>2017</number>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="nameLabel">
       <property name="text">
        <string>Название:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="pagesLabel">
       <property name="text">
        <string>Количество страниц:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="copiesLabel">
       <property name="text">
        <string>Количество экземпляров:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QSpinBox" name="copiesSpinBox">
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="maximum">
        <number>1000</number>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="yearLabel">
       <property name="text">
        <string>Год издания:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLabel" name="copiesMUST">
       <property name="styleSheet">
        <string notr="true">color : red;</string>
       </property>
       <property name="text">
        <string notr="true">*</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLabel" name="pagesMUST">
       <property name="styleSheet">
        <string notr="true">color : red;</string>
       </property>
       <property name="text">
        <string notr="true">*</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="yearMUST">
       <property name="styleSheet">
        <string notr="true">color : red;</string>
       </property>
       <property name="text">
        <string notr="true">*</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QPushButton" name="editAuthorsButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="Resouces.qrc">
         <normaloff>:/resource/icons/pencil.png</normaloff>:/resource/icons/pencil.png</iconset>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="authorMUST">
       <property name="styleSheet">
        <string notr="true">color : red;</string>
       </property>
       <property name="text">
        <string notr="true">*</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="ISBNMUST">
       <property name="styleSheet">
        <string notr="true">color : red;</string>
       </property>
       <property name="text">
        <string notr="true">*</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLineEdit" name="ISBNText">
       <property name="maxLength">
        <number>17</number>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="ISBNLabel">
       <property name="text">
        <string>ISBN номер:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLineEdit" name="nameText">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLineEdit" name="authorsText">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>ISBNText</tabstop>
  <tabstop>editAuthorsButton</tabstop>
  <tabstop>nameText</tabstop>
  <tabstop>yearSpinBox</tabstop>
  <tabstop>pagesSpinBox</tabstop>
  <tabstop>copiesSpinBox</tabstop>
  <tabstop>editInfoButton</tabstop>
  <tabstop>archieveBookButton</tabstop>
  <tabstop>renewBookButton</tabstop>
  <tabstop>getBackBookButton</tabstop>
  <tabstop>bookInfoTable</tabstop>
 </tabstops>
 <resources>
  <include location="Resouces.qrc"/>
 </resources>
 <connections/>
</ui>
